### ISSUE TITLE
Nether Star Generators

### ISSUE THREAD

**Issue by Piddu** [rwtema/extrautilities_old_issues#439]
I've been doing some tests with these generators and the latest patch, i just switched from the 1.1.0K version.

Starting from the normal generator. with 1 star it produces 40.000 RF/t for 120 seconds, this makes:

40.000 RF \* 20 tick \* 120 seconds = 96.000.000 RF

In 1.1.0k version with a 8x generator i've got the same total RF and it takes 1/8 of time, 1 star lasts 15 sec.
In latest version a 8x generator produces 30.000.000 rf with one star.
I've connected the generator directly to an EnderIO capacitor with 2BIlion total capacity and a buffer of 2M. Still does 30M total RF.
I've even tried to connect it through a tesserac, still same result.
Also tried to connect it to the capacitor with 5  Hyper Energy Transfer Nodes

So i've also tried the 64x , 1 star produces nearly 3 Millions RF. 

In conclusion unlikely other generators and their respectively tiers were the total amount of energy X fuel unit it's the same, but they burn in 1/8 or 1/64 of time, the Nether Star Generators still burn the fuel faster, but the amount of energy produced it's not the same.

Is this working as intend or have i misunderstood or missed something?

**Comment by Frontrider** [comment]
The  fastest nether star generator roduces more rf than its buffer.

**Comment by Piddu** [comment]
yea it's like the 8x and 64x transfer rates are capped to an X value per tick, and that makes them produce less energy than the 1x generator ...
